etc/config/.env.example

```diff
@@ -42,8 +42,9 @@ BROWSER=chrome
 MODULE_WHITELIST=Magento_Framework,Magento_ConfigurableProductWishlist,Magento_ConfigurableProductCatalogSearch
 #CUSTOM_MODULE_PATHS=
 
-#*** Bool property which allows the user to toggle debug output during test execution
+#*** Bool properties which allows the user to toggle debug output during test execution
 #MFTF_DEBUG=
+#MFTF_FAST_DEBUG=
```

Comment:
What else is `MFTF_DEBUG` used for, and why don't we just repurpose the old flag instead of adding a new one?
I don't see the point in adding `FAST_DEBUG` if noone will even want ot use `MFTF_DEBUG` flag to begin with.

Reply:
This flag is removed - enabled merged file debugging for generate:tests by default